pkgs/development/python-modules/latex2mathml/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildPythonPackage, fetchFromGitHub, poetry-core
+, multidict, pytestCheckHook, pytest-cov, xmljson }:
+
+buildPythonPackage rec {
+  pname = "latex2mathml";
+  version = "3.75.2";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "roniemartinez";
+    repo = pname;
+    rev = version;
+    hash = "sha256-i/F1B/Rndg66tiKok1PDMK/rT5c2e8upnQrMSCTUzpU=";
+  };
+
+  nativeBuildInputs = [ poetry-core ];
+  nativeCheckInputs = [ multidict pytestCheckHook pytest-cov xmljson ];
+
```

Comment:
pythonImportsCheck = [ "latex2mathml" ];

Reply:
It should already be done in test phase I think.

Reply:
I don't think it is, if you compare the build logs you should see `Check whether the following modules can be imported: latex2mathml` with the argument set, it doesn't happen without it.

Reply:
No, I mean the test suite already import it in order to test it.  IMHO `pythonImportsCheck` is a smoke test and only useful when there's no test.